SpeechToText: Add API descriptions (#8048)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxSpeechToText/1 Configuration/onOptionChanged.md

---
---
##### shortDescription
A function that is executed after a UI component property is changed.

##### param(e): ui/speech_to_text:OptionChangedEvent
Information about the event.

##### field(e.component): {WidgetName}
The UI component's instance.

##### field(e.element): DxElement
#include common-ref-elementparam with { element: "UI component" }

##### field(e.fullName): String
The path to the modified property that includes all parent properties.

##### field(e.name): String

> The modified property if it belongs to the first level. Otherwise, the first-level property in which it is nested.

Reviewer: ```suggestion
The modified property. For nested properties, returns the root-level property.
```
